aries-site/src/layouts/main/Layout.js

```diff
@@ -53,15 +66,51 @@ export const Layout = ({
   const layout = isLanding ? 'plain' : pageLayout;
 
   const MainContentWrapper = isLanding ? Fragment : PageContent;
-  const size = useContext(ResponsiveContext);
+  const breakpoint = useContext(ResponsiveContext);
+  const [open, setOpen] = useState(false);
+  const onOpen = () => setOpen(true);
+  const onClose = () => setOpen(undefined);
+  const [value, setValue] = useState();
+
+  const closeFeedbackModal = () => {
+    setTimeout(() => {
+      setOpen(false);
+    }, 2000);
+  };
+
+  const onSubmit = useCallback(() => {
```

Comment:
I think you could get rid of the `onChange` and `value` useState and take the value handed to the onSubmit instead (if you also made the change in the form I mentioned earlier.)
```suggestion
  const onSubmit = useCallback(({ value }) => {
```